💡 style: switched to Ariakit's tooltip

berry-13 opened the pull request:
## Summary

moved to ariakit's tooltip

## Change Type

- [x] Style

## Checklist

- [x] My code adheres to this project's style guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented in any complex areas of my code
- [x] I have made pertinent documentation changes
- [x] My changes do not introduce new warnings
- [x] I have written tests demonstrating that my changes are effective or that my feature works
- [x] Local unit tests pass with my changes
- [x] Any changes dependent on mine have been merged and published in downstream modules.
- [x] A pull request for updating the documentation has been submitted.

berry-13 committed:
inital Tooltip implementation and test

berry-13 committed:
style(tooltip): L/R sidePanel and Nav

berry-13 committed:
style(tooltip): unarchive button; refactor: `useArchiveHandler` and `ArchiveButton`

berry-13 committed:
style(tooltip): Delete button

berry-13 committed:
refactor: remove unused className prop in DeleteButton component

berry-13 committed:
style(tooltip): finish final tooltip and fix bookmark edit and delete button

berry-13 committed:
refactor(ui): remove TooltipTest and DropDownMenu component and unused imports

berry-13 committed:
style: update mobile UI

danny-avila commented:
On first login, the tooltips are no longer rendered, but the bigger issue is the right-hand side panel no longer shows icons (selected/highlighted blue text)

![image](https://github.com/user-attachments/assets/cc3cc4b6-da8f-45cc-affb-b33531ae82bc)

berry-13 committed:
fix: sidePanel icon not showing

berry-13 committed:
feat(AttachFile): add tooltip

berry-13 committed:
fix(NavToggle): remove button
without this button, kb users don't have to manually press 2 times to change the focus
Also, tooltips with buttons focus don't trigger

berry-13 committed:
fix: right side panel issue with double button

danny-avila commented:
Sorry this conflicts with my menu PR. This is not a high priority item and I would wait till my dev build is in. Please remind me to merge this after

berry-13 committed:
fix: merge issues

berry-13 committed:
fix: sharedLink table issue

berry-13 commented:
@danny-avila fixed all merge conflicts caused by #3864

berry-13 committed:
chore: update ariakit and framer-motion version

berry-13 committed:
a11y: kb toggle for sidebar

berry-13 committed:
feat: tooltip for some buttons
